packages/editor-ui/src/components/NodeCreator/NodeItem.vue

```diff
@@ -1,15 +1,23 @@
-<template functional>
-	<div :class="{[$style['node-item']]: true, [$style.bordered]: props.bordered}">
-		<NodeIcon :class="$style['node-icon']" :nodeType="props.nodeType" :style="{color: props.nodeType.defaults.color}" />
+<template>
+	<div :class="{[$style['node-item']]: true, [$style.bordered]: bordered}">
+		<NodeIcon :class="$style['node-icon']" :nodeType="nodeType" :style="{color: nodeType.defaults.color}" />
 		<div>
 			<div :class="$style.details">
-				<span :class="$style.name">{{props.nodeType.displayName}}</span>
+				<span :class="$style.name">
+          {{translateSpecific({
+              key: `${nodeType.name}.displayName`,
+              fallback: nodeType.displayName,
+          })}}
+        </span>
 				<span :class="$style['trigger-icon']">
-					<TriggerIcon v-if="$options.isTrigger(props.nodeType)" />
+					<TriggerIcon v-if="nodeType.group.includes('trigger')" />
 				</span>
 			</div>
 			<div :class="$style.description">
-				{{props.nodeType.description}}
+				{{translateSpecific({
+            key: `${nodeType.name}.description`,
+            fallback: nodeType.description,
+        })}}
```

Comment:
See comment above for `translateSpecific`.

Reply:
Fixed, thanks!